.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+indent_style = space
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+max_line_length = 100
+indent_size = 4
```

Comment:
I'm not sure if there's a better way to add uniformity with indentations with a Haskell projects. If it's too opinionated, I can remove it from this changeset if you'd like.

Reply:
i've been using hindent to format the haskell code - i don't think this file is needed.